packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6030,7 +6025,10 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   private requestScreenShareFloor() {
-    if (!this.mediaProperties.shareVideoStream || !this.mediaProperties.mediaDirection.sendShare) {
+    if (
+      this.mediaProperties.hasLocalShareStream() ||
```

Comment:
this is the opposite of what it should be, it should be `!this.mediaProperties.hasLocalShareStream()` - this should be caught by unit tests for requestScreenShareFloor like this one [here](https://github.com/webex/webex-js-sdk/blob/b58650c19f200e888239d496f2557d443b945f0b/packages/%40webex/plugin-meetings/test/unit/spec/meeting/index.js#L2557). Is it passing? if it is, can you check why?

Reply:
You're right! The test did fail. Have fixed it now. Thanks.